.devcontainer/Dockerfile

```diff
@@ -13,12 +13,9 @@ RUN groupadd --gid $USER_GID $USERNAME \
     && chmod 0440 /etc/sudoers.d/$USERNAME
 
 RUN apt update && \
-    curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - && \
-    apt update && \
-    apt install -y --no-install-recommends nodejs libssl-dev musl-dev make perl curl 
+    apt install -y --no-install-recommends libssl-dev musl-dev make perl curl gzip
```

Comment:
We can throw in a `&& rm -rf /var/lib/apt/lists/*` for good measure, to reduce the image size. Probably not very important, but still :)

Reply:
done (though it didn't make a visible different in the 1.94gb image size)